Fix fs drivers build warnings since roundup() defined globally at kernel.h now

diff -urB linux-2.4.37.orig/fs/xfs/linux-2.4/xfs_linux.h linux/fs/xfs/linux-2.4/xfs_linux.h
--- linux-2.4.37.orig/fs/xfs/linux-2.4/xfs_linux.h	2009-04-29 19:48:17.000000000 +0400
+++ linux/fs/xfs/linux-2.4/xfs_linux.h	2009-04-29 19:47:13.000000000 +0400
@@ -252,7 +252,6 @@
 #define MIN(a,b)	(min(a,b))
 #define MAX(a,b)	(max(a,b))
 #define howmany(x, y)	(((x)+((y)-1))/(y))
-#define roundup(x, y)	((((x)+((y)-1))/(y))*(y))
 
 /*
  * Juggle IRIX device numbers - still used in ondisk structures
diff -urBp linux-2.4.37.orig/fs/binfmt_elf.c linux/fs/binfmt_elf.c
--- linux-2.4.37.orig/fs/binfmt_elf.c	2005-05-23 16:12:31.000000000 +0400
+++ linux/fs/binfmt_elf.c	2009-06-12 10:39:57.000000000 +0400
@@ -1076,8 +1076,6 @@ static inline int maydump(struct vm_area
 	return 1;
 }
 
-#define roundup(x, y)  ((((x)+((y)-1))/(y))*(y))
-
 /* An ELF note in memory */
 struct memelfnote
 {
diff -urBp linux-2.4.37.orig/fs/proc/kcore.c linux/fs/proc/kcore.c
--- linux-2.4.37.orig/fs/proc/kcore.c	2009-06-07 23:50:24.000000000 +0400
+++ linux/fs/proc/kcore.c	2009-06-12 10:40:46.000000000 +0400
@@ -103,8 +103,6 @@ static ssize_t read_kcore(struct file *f
 }
 #else /* CONFIG_KCORE_AOUT */
 
-#define roundup(x, y)  ((((x)+((y)-1))/(y))*(y))
-
 /* An ELF note in memory */
 struct memelfnote
 {
